### PR TITLE
Added experimental json.parse function

### LINKED ISSUE #1072: Document 'json.parse' function
PR: https://github.com/influxdata/flux/pull/2860

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1072 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/reference/flux/stdlib/experimental/json/_index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+menu:
+  v2_0_ref:
+    name: JSON
+    identifier: JSON-exp
+    parent: Experimental
+weight: 301
+v2.0/tags: [functions, json, package]
+---
+
+Flux JSON functions provide tools for working with Message Queuing Telemetry Transport (JSON) protocol.
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/json/_index.md:16]: (MQTT) protocol...JSON data?

File: content/v2.0/reference/flux/stdlib/experimental/json/parse.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  The `json.parse()` function takes JSON data as bytes and returns a value.
+menu:
+  v2_0_ref:
+    name: json.parse
+    parent: JSON-exp
+weight: 401
+---
+
+The `json.parse()` function takes JSON data as bytes and returns a value.
+The function can product lists, objects, strings, booleans, and float values.
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/json/parse.md:13]: can return?

File: content/v2.0/reference/flux/stdlib/experimental/json/parse.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+### data
+JSON data to parse.
+
+_**Data type:** Bytes_
+
+
+## Examples
+
+##### Use JSON data to restructure a table
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/json/parse.md:36]: Parse JSON data to restructure a table?

File: content/v2.0/reference/flux/stdlib/experimental/json/parse.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+      return {
+        _time: r._time,
+        _field: r._field,
+        a: jsonData.a,
+        b: jsonData.b,
+        c: jsonData.c,
+      }
+    })
+  |> keep(columns: ["_time", "_field", "a", "b", "c"]))
```
Comment (nathanielc) [inline content/v2.0/reference/flux/stdlib/experimental/json/parse.md:52]: I would remove the keep as its redundant at this point. (we did the same thing to the test case)

**Pull request merged**